tests/apps/lims/test_limsjson.py

```diff
@@ -25,7 +25,6 @@ def test_parsing_balsamic_json(balsamic_order_to_submit):
 
     assert sample["name"] == "s1"
     assert sample["container"] == "96 well plate"
-    assert sample["data_analysis"] == str(Pipeline.BALSAMIC)
```

Comment:
Why remove this one. Seems like the rest of tests expect data_analysis. But why on sample level?

Reply:
Yeah, I don't remember.. I'll reintroduce it and test for data_delivery as well :)